packages/next/src/build/webpack/config/blocks/base.ts

```diff
@@ -2,6 +2,19 @@ import curry from 'next/dist/compiled/lodash.curry'
 import type { webpack } from 'next/dist/compiled/webpack/webpack'
 import { COMPILER_NAMES } from '../../../../shared/lib/constants'
 import type { ConfigurationContext } from '../utils'
+import DevToolsIgnorePlugin from '../../plugins/devtools-ignore-list-plugin'
+import EvalSourceMapDevToolPlugin from '../../plugins/eval-source-map-dev-tool-plugin'
+
+function shouldIgnorePath(modulePath: string): boolean {
+  // TODO: How to ignore list 'webpack:///../../../src/shared/lib/is-thenable.ts'
+  return (
+    modulePath.includes('node_modules') ||
+    // would filter 'webpack://_N_E/./app/page.tsx'
+    // modulePath.startsWith('webpack://_N_E/') ||
+    // e.g. 'webpack:///external commonjs "next/dist/compiled/next-server/app-page.runtime.dev.js"'
+    modulePath.includes('next/dist')
```

Comment:
can we also check if it's starting with `next/dist` or there's a slash at the beginning? like the regext `\/?next\/dist`

Reply:
> can we also check if it's starting with next/dist or there's a slash at the beginning? like the regext \/?next\/dist

What would that do? 

This is also an outdated version. We really just need this when `next` is symlinked i.e. local development.

Reply:
techinically it wil match sth `foo-next/dist`, but i realize it would also match `my-lib/next/dist`. If we can check if it's including the path of nextjs package would be more ideal but not critical for now